### DISCUSSION TITLE
What are the units of stoichiometric coefficients of DNA polimerization reaction?

### DISCUSSION THREAD

**Discussion by PkiwiBird** [SysBioChalmers/Human-GEM#522] [Q&A] [answered]
Human-GEM has a lumped biomass reaction (MAR13082), so the stoichiometric coefficient of DNA in that reaction is probably mmol of DNA/gDW, but what are the units of the trinucleotides (like dCTP) in DNA polymerization reaction [MAR07160](https://metabolicatlas.org/explore/Human-GEM/gem-browser/reaction/MAR07160)?
mols of trinucleotide or mmol of trinucleotide/g of DNA?
I ask because I thought the stoichiometric coefficients are usually mols in "normal" reactions (non-pseudo reactions).

**Comment by haowang-bioinfo** [comment, marked as answer]
> what are the units of the trinucleotides (like dCTP) in DNA polymerization reaction [MAR07160](https://metabolicatlas.org/explore/Human-GEM/gem-browser/reaction/MAR07160)?

the unit is mol/mol

**Reply by PkiwiBird** [reply]
So the units of coefficients of metabolites of all reactions except biomass reaction are mol/mol and biomass is mmol/gDW. Ok, thank you!
